source/documentation/tools/package-management.md

```diff
@@ -1,87 +1,4 @@
-
-* which project and files are open (displayed in the RStudio interface)
-* console and terminal history
-* the global environment – R variables that have been set and are held in memory
-
-The session is persisted between restarts of RStudio, to make it easier for you to pick up where you left off. However you can clear it to solve a number of problems, including if the memory is full.
-
-To clear the RStudio session:
-
-1. Close RStudio, if it is open in any window (because it continually saves its session to disk).
-
-2. Open JupyterLab.
-
-3. In JupyterLab, open a Terminal and run:
-
-    ```bash
-    rm -rf ~/.rstudio.bak; mv ~/.rstudio/ ~/.rstudio.bak
-    ```
-
-4. In the control panel, select the __Restart__ button for RStudio.
-
-    ![RStudio's "Restart" button in Control Panel](images/tools/restart_rstudio.png)
-
-5. In the control panel, select __Open__ for RStudio. It may take between one and five minutes before RStudio is available. You may need to refresh your browser for the tool to load.
-
-## Package management
-
-A key element of ensuring analysis is reproducible is maintaining a record of the versions of packages used to produce said analysis.
-
-There are a number of tools available for both R and Python to support package management.
-
-For R:
-
-* [**conda**](#conda) - Recommended for R in Analytical Platform. (Conda is only provided on AP with RStudio images which have R 3.5.1 and later. So if you have the older RStudio with R 3.4 then you'll need to use Packrat instead.)
-* [**packrat**](#packrat) - not recommended for R, but remains an option.
-
-For Python:
-
-* [**venv** and **pip**](#venv-and-pip) - Recommended for Python in Analytical Platform, because it's easier, more reliable and has a much bigger community than conda. **NOTE:** To use pip with venv, you may need to delete the `.bash_aliases` file (`rm .bash_aliases`) in your home directory.
-* [**conda environment**](#conda) installing packages with just **conda** - not recommended, but it might help for the occasional package whose C extension doesn't install well with pip (perhaps the pip package isn't the newer 'wheel' type, or it doesn't have a binary suitable for our distribution)
-* [**conda environment**](#conda) installing packages with **conda** and **pip** - not recommended, but gives you the broadest range of package install options. However conda and pip don't play well together - use at your own risk!
-
-**Note on support:** the Analytical Platform team does not offer support on the topic of packaging. DASD users are encouraged to use their #r, #conda and #python Slack channels to support each other, or ask your line manager about training. Of course, if there is something broken with the platform itself, or something unique about the platform that prevents you from installing a library, then of course do raise it with the team in the normal ways.
+# Package Management
 
 ## Conda
```

Comment:
I think it would be useful to have a brief paragraph at the start of this section that lists the various package management options and where each one should be used.